core-bundle/src/Migration/Version503/RememberMeMigration.php

```diff
@@ -46,17 +46,20 @@ public function run(): MigrationResult
                 SQL,
         );
 
+        $schemaManager = $this->connection->createSchemaManager();
+        $username = isset($schemaManager->listTableColumns('tl_remember_me')['userIdentifier']) ? 'userIdentifier' : 'username';
```

Comment:
```suggestion
        $username = isset($schemaManager->listTableColumns('tl_remember_me')['useridentifier']) ? 'userIdentifier' : 'username';
```

`listTableColumns` returns the column names in lower case, does it not?